tests/drivers/dma/chan_link_transfer/testcase.yaml

```diff
@@ -0,0 +1,13 @@
+tests:
+  drivers.dma:
+    min_ram: 16
+    depends_on: dma
+    tags: drivers dma
+    platform_allow: frdm_k64f mimxrt1060_evk
```

Comment:
is this really NXP specific? Can we filter on something else here instead to use the same test on other platforms that have the support?

Reply:
@nashif , the added feature is from NXP EDMA feature of minor / major loops by using the dest_chaining_en and src_chaining_en, which is related to the RM in below:
![image](https://user-images.githubusercontent.com/800198/105656215-aea8ce00-5efc-11eb-8834-cb3c9a0db27d.png)

I did not see any other EDMA drivers defines likely functions. the dest_chaining_en is only defined in the dma.h before. never used. looks like the dma driver is marked as orphaned so shall I raise it to API meeting to get unified?